plugins/oracle-bulk-loader/impl/src/main/java/org/pentaho/di/trans/OraBulkDataOutput.java

```diff
@@ -20,7 +20,7 @@
  *
  ******************************************************************************/
 
-package org.pentaho.di.trans.steps.orabulkloader;
+package org.pentaho.di.trans;
```

Comment:
Why did we need to move this into a different package?

Reply:
changed the packaged and recommited